### PR TITLE
[Fix/#45]: 사파리 css 오류 수정, 토큰 저장 방식 수정 등

### PR DESCRIPTION
**Pull request by JIMIN1020**
- 사파리 css 오류 수정
- 토큰 저장 방식 수정 (임시 sessionStorage)
- 공동구매 요청/취소 버튼 스타일링 변경
- axios instance 변경

### ✔️ axios instance
- baseInstance: token이 필요없는 요청인 경우 - interceptor가 적용되어 있지 않습니다 (로그인, 회원가입, 카탈로그 조회 등)
- authInstance: token이 필요한 경우 - interceptor 적용 (프로필, 카탈로그 상세 등)

auth instance는 요청 전에 토큰을 헤더에 추가하는 로직이 포함되어 있습니다.
```js
authInstance.interceptors.request.use(
  (req) => {
    const { accessToken } = getTokenFromStorage();
    if (accessToken) {
      req.headers.Authorization = accessToken;
    }
    return req;
  },
  (err) => Promise.reject(err)
);
```

### ✔️ 토큰 저장방식 수정
토큰을 body로 전달받아 지금은 sessionStorage에 저장해두었습니다.
(추후 자동로그인 구현 시 상황에 따라 localStorage에 저장)

<br/>

## 📌 구현 결과 (선택)
cs

<br/>

## 📌 기타 사항
- useSessionStore는 로그인 여부(isLoggedIn)를 저장해두는 스토어였는데, sessionStorage에 토큰을 저장하면 이걸로 로그인 여부를 판별해도 괜찮을까요? 항상 고민됐던 부분이라 다른 팀원분들 생각이 궁금합니다!

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">0%</td> <td align="right">0 / 3736</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">0%</td> <td align="right">0 / 3736</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">2.08%</td> <td align="right">3 / 144</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">2.08%</td> <td align="right">3 / 144</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/auth.api.ts">src/api/auth.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/auth.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/auth.api.ts#L3-L10" class="text-red">3-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/auth.api.ts#L12-L22" class="text-red">12-22</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts">src/api/instance.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L7-L14" class="text-red">7-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L16-L23" class="text-red">16-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L25-L34" class="text-red">25-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L36-L42" class="text-red">36-42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L44-L51" class="text-red">44-51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L53" class="text-red">53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/api/instance.ts#L55-L70" class="text-red">55-70</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx">src/components/Button/Button.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L14-L32" class="text-red">14-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L34-L35" class="text-red">34-35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L37" class="text-red">37</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L39" class="text-red">39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L41-L43" class="text-red">41-43</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L53-L57" class="text-red">53-57</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L61-L78" class="text-red">61-78</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L81-L87" class="text-red">81-87</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L89-L104" class="text-red">89-104</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Button/Button.tsx#L106-L126" class="text-red">106-126</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx">src/components/Form/AuthForm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L1-L7" class="text-red">1-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L9-L17" class="text-red">9-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L19-L34" class="text-red">19-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L36-L62" class="text-red">36-62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L64" class="text-red">64</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L66-L73" class="text-red">66-73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L75-L78" class="text-red">75-78</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L80-L87" class="text-red">80-87</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L89" class="text-red">89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L91" class="text-red">91</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L93" class="text-red">93</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L100" class="text-red">100</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L110" class="text-red">110</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/AuthForm.tsx#L118-L119" class="text-red">118-119</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx">src/components/Form/PasswordInput.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L6-L8" class="text-red">6-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L10-L13" class="text-red">10-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L15-L47" class="text-red">15-47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L49" class="text-red">49</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L51" class="text-red">51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L53" class="text-red">53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L66-L69" class="text-red">66-69</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Form/PasswordInput.tsx#L76-L77" class="text-red">76-77</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx">src/components/Header/Header.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L7-L20" class="text-red">7-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L22" class="text-red">22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L24" class="text-red">24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L26" class="text-red">26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L51" class="text-red">51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L53" class="text-red">53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/components/Header/Header.tsx#L56-L57" class="text-red">56-57</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/constants/storage.ts">src/constants/storage.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/constants/storage.ts#L1-L4" class="text-red">1-4</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/liquor.api.ts">src/features/CatalogDetail/api/liquor.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/liquor.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/liquor.api.ts#L3-L10" class="text-red">3-10</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/register.api.ts">src/features/CatalogDetail/api/register.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/register.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/register.api.ts#L3-L6" class="text-red">3-6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/api/register.api.ts#L9-L17" class="text-red">9-17</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx">src/features/CatalogDetail/components/SideBar.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L1-L6" class="text-red">1-6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L8-L11" class="text-red">8-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L13" class="text-red">13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L15" class="text-red">15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L17-L27" class="text-red">17-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L29-L30" class="text-red">29-30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L32" class="text-red">32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L34-L36" class="text-red">34-36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L40" class="text-red">40</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L42" class="text-red">42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/CatalogDetail/components/SideBar.tsx#L52" class="text-red">52</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/FindPassword/api/findPassword.api.ts">src/features/FindPassword/api/findPassword.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/FindPassword/api/findPassword.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/FindPassword/api/findPassword.api.ts#L3-L11" class="text-red">3-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/FindPassword/api/findPassword.api.ts#L13-L17" class="text-red">13-17</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts">src/features/Join/api/signup.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L7-L9" class="text-red">7-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L11-L14" class="text-red">11-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L16-L18" class="text-red">16-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L20-L23" class="text-red">20-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Join/api/signup.api.ts#L25-L34" class="text-red">25-34</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/api/login.api.ts">src/features/Login/api/login.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/api/login.api.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/api/login.api.ts#L6-L15" class="text-red">6-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/api/login.api.ts#L17-L23" class="text-red">17-23</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts">src/features/Login/hooks/useLogin.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts#L6" class="text-red">6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts#L8-L11" class="text-red">8-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts#L13-L24" class="text-red">13-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/hooks/useLogin.ts#L26-L27" class="text-red">26-27</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Login/models/user.model.ts">src/features/Login/models/user.model.ts</a></td> <td align="right">100%</td> <td align="right">100%</td> <td align="right">100%</td> <td align="right">100%</td> <td align="left"></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx">src/features/Main/components/BrandNew/BrandNew.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx#L5-L12" class="text-red">5-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx#L14" class="text-red">14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx#L16" class="text-red">16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Main/components/BrandNew/BrandNew.tsx#L35" class="text-red">35</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/getProfile.ts">src/features/Profile/api/getProfile.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/getProfile.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/getProfile.ts#L3-L6" class="text-red">3-6</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/putProfile.ts">src/features/Profile/api/putProfile.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/putProfile.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/features/Profile/api/putProfile.ts#L4-L7" class="text-red">4-7</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx">src/layouts/AuthLayout.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L6-L18" class="text-red">6-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L20" class="text-red">20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L22" class="text-red">22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L24" class="text-red">24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L32" class="text-red">32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/layouts/AuthLayout.tsx#L46" class="text-red">46</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/utils/storage.ts">src/utils/storage.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/utils/storage.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/utils/storage.ts#L3-L8" class="text-red">3-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/fb866da3311869247d4f46106e466d7f5d415a3d/src/utils/storage.ts#L10-L15" class="text-red">10-15</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10279408295>#59</a></em>

**Review by SebellKo — APPROVED** [review]
지민님 고생하셨습니다..!!

loginStore는 페이지 권한을 위한, 또 토큰은 요청을 위해 저장하는거죠 ?!
음,, 저도 지민님 생각대로 토큰으로 로그인 판별이 가능하니 loginStore는 불필요할것 같다고 생각이 들면서도... 하지만 로그인 판별 확인과, 요청을 위한 2개의 목적으로 사용한다면 두가지를 나누는게 맞는것 같기도 하고...

저는 loginStore도 유지하는게 좋은것 같아요 !

File: src/api/instance.ts
Comment (juhee067) [inline src/api/instance.ts:1]: 상황에 따라 instance를 분리 한다는 건 몰랐네요👍🏻👍🏻👍🏻

**Review by juhee067 — APPROVED** [review]
지민님 너무 고생하셨어요!! 👍🏻
지금 수정한 css가 다른 브라우저에서도 오류가 생기지 않는 것일까요? 사파리에서만 수정이 되는 것일까요?!

**Comment by JIMIN1020** [comment]
> 지민님 너무 고생하셨어요!! 👍🏻 지금 수정한 css가 다른 브라우저에서도 오류가 생기지 않는 것일까요? 사파리에서만 수정이 되는 것일까요?!

chrome, safari, firefox까지 확인했습니다!

File: src/api/instance.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        if (newAccessToken) {
+          sessionStorage.setItem(STORAGE_KEY.accessToken, newAccessToken);
+          originRequest.headers.Authorization = newAccessToken;
+        }
+        return authInstance(originRequest);
+      } catch (error) {
+        setIsLoggedIn(false);
+        deleteTokenFromStorage();
+        window.alert('로그인 세션이 만료되었습니다. 다시 로그인해주세요.');
```
Comment (ksy90101) [inline src/api/instance.ts:63]: 이부분은 토스트로 보여주는게 더 좋을거 같긴해요

File: src/components/Form/PasswordInput.tsx [hunk 2]
```diff
@@ -35,13 +35,13 @@ function PasswordInput() {
       />
       <div className='requirements'>
         <Requirement $valid={hasEng()}>
-          <CheckIcon /> 영문
+          <CheckIcon width={18} height={18} /> 영문
```
Comment (ksy90101) [inline src/components/Form/PasswordInput.tsx:38]: Icon에
차라리 size를 하는게 더 좋을거 같긴합니다 svg아이콘은 같은 높이외
넓이를 쓰긴해서여

File: src/features/FindPassword/api/findPassword.api.ts [hunk 3]
```diff
@@ -1,17 +1,17 @@
-import { httpClient } from '@/api/http';
+import { baseInstance } from '@/api/instance';
 
 export const requestResetPassword = async (
   companyNumber: string,
   contact: string
 ) => {
-  await httpClient.post('/users/password-reset', {
+  await baseInstance.post('/users/password-reset', {
```
Comment (ksy90101) [inline src/features/FindPassword/api/findPassword.api.ts:7]: 인스턴스 차이가 뭐에여? 이거 내일 허들로 한번 이야기하면 좋을거 같어요

**Pull request merged**